profiles/elasticsearch/terraform/elasticsearch/variables.tf.ctmpl

```diff
@@ -0,0 +1,119 @@
+#
+# Profile Vars
+#
+variable "google_project" {
+  default = "{{env "GOOGLE_PROJECT"}}"
+  description = "The google project as specified in the application json"
+}
+variable "owner" {
+  default = "{{env "OWNER"}}"
+  description = "The owner from the application json"
+}
+variable "application_name" {
+  default = "{{env "APPLICATION_NAME"}}"
+  description = "Name from the application json"
+}
+variable "service" {
+  default = "{{env "SERVICE"}}"
+  description = "The name of the service within the profile"
+}
+variable "vault_path_prefix" {
+  default = "{{env "VAULT_PATH_PREFIX"}}"
+  description = "The prefix path at which to store vault values"
+}
+
+
+#
+# Dedency Profiles' Vars
+#
+# DNS
+variable "dns_zone_name" {
+  default = "{{env "DNS_ZONE_NAME"}}"
+  description = "The name of the DNS zone (its google id) as specified in the application json"
+}
+variable "dns_project" {
+  default = "{{if (env "TERRA_DNS_PROJECT")}}{{env "TERRA_DNS_PROJECT"}}{{else}}broad-dsde-perf{{end}}"
+}
+variable "dns_region" {
+  default = "{{if (env "TERRA_DNS_REGION")}}{{env "TERRA_DNS_REGION"}}{{else}}us-central1{{end}}"
+}
+# SSL
+variable "google_compute_ssl_certificate_red" {
+  default = "{{env "OWNER"}}-terra-env-wildcard-ssl-certificate-red"
+}
+variable "google_compute_ssl_certificate_black" {
+  default = "{{env "OWNER"}}-terra-env-wildcard-ssl-certificate-black"
+}
+data "google_compute_ssl_certificate" "terra-env-wildcard-ssl-certificate-red" {
+  name = "${var.google_compute_ssl_certificate_red}"
+}
+data "google_compute_ssl_certificate" "terra-env-wildcard-ssl-certificate-black" {
+  name = "${var.google_compute_ssl_certificate_black}"
+}
+# Network
+variable "google_network_name" {
+  default = "{{env "OWNER"}}-terra-network"
+}
+data "google_compute_network" "terra-env-network" {
+  name = "${var.google_network_name}"
+}
+# SA
+variable "config_reader_service_account" {
+  default = "{{env "OWNER"}}-{{env "SERVICE"}}"
+}
+data "google_service_account" "config_reader" {
+  account_id = "${var.config_reader_service_account}"
+}
+
+
+#
+# - Common Vars
+#
+variable "instance_tags" {
+  default = [
+    "{{env "OWNER"}}-terra-{{env "SERVICE"}}",
+    "http-server",
+    "https-server",
+    "{{env "SERVICE"}}",
```

Comment:
Add "mongo"?

Reply:
this is elasticsearch, the service name is `elasticsearch`